tests/unit/objects/test_merge_requests.py

```diff
@@ -40,6 +40,7 @@
             "web_url": "http://gitlab.example.com//kenyatta_oconnell",
         }
     ],
+    "related_issues": [1, 2],
```

Comment:
In reality, GitLab will respond with a complete JSON object with more details of each issue. See the example response in https://docs.gitlab.com/ee/api/merge_requests.html#list-issues-related-to-the-merge-request.

Additionally, as this is an entire new endpoint we'll need to actually mock the `/related_issues` response separately from the `resp_list_merge_requests`. Let me know if you need any help with that :bow: